docs/users-guide.md

```diff
@@ -1,3 +1,78 @@
 # OpenShift Service Mesh Console User's Guide
 
-TODO
+The OpenShift Service Mesh Console (aka OSSMC) is an extension to the OpenShift Console which provides visibility into your Service Mesh. The functionality you see described here is very similar to that of the standalone Kiali Console (see https://kiali.io). In fact, you can still access the standalone Kiali Console if you wish. This User's Guide, however, will discuss the functionality that you see from within the OpenShift Console itself.
+
+:warning: The OSSMC [only supports a single tenant today](https://github.com/kiali/openshift-servicemesh-plugin/issues/28). Whether that tenant is configured to access only a subset of OpenShift projects or has access cluster-wide to all projects does not matter, however, only a single tenant can be accessed.
+
+# Overview
```

Comment:
Do you need a screenshot for the overview page as well?

Reply:
Screenshot of the overview page is the first one in the user's guide: https://github.com/kiali/openshift-servicemesh-plugin/blob/eed0a8eab662ab476adc265d22bcee4e12ba315c/docs/users-guide.md#overview

This: https://github.com/kiali/openshift-servicemesh-plugin/blob/eed0a8eab662ab476adc265d22bcee4e12ba315c/docs/20-overview.png